### PR TITLE
Add information on OCI

### PR DESCRIPTION
**Pull request by longshorej**
This adds some information on creating OCI bundles and mentions OCI considerations for installation, release notes.

### CODE REVIEW

**Comment by longshorej** [comment]
[Document OCI enhancements, being sure to link to doco from the release notes page](https://trello.com/c/2cGP9YRu/596-document-oci-enhancements-being-sure-to-link-to-doco-from-the-release-notes-page)

File: src/main/play-doc/developer/CreatingBundles.md [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+If you'd rather save the resulting bundle to a file, you can do this using `bndl -o <filename>`.
+```bash
+docker save my-image | bndl -o my-bundle.zip
+```
+
+
+We attempt to extract as much information from the your docker image as possible. Any `EXPOSE` directives in your `Dockerfile` will automatically create corresponding service endpoints in `bundle.conf`. Support for persistent mounting of `VOLUME` directives is planned.
+
+Docker images can vary in size significantly and this can affect development/deployment time and consume additional system resources. Because of this, we recommend that you build your images ontop of a light-weight base image. [Alpine](https://hub.docker.com/_/alpine/) is a great choice as it's only 5MB in size. 
+
```
Comment (huntc) [inline src/main/play-doc/developer/CreatingBundles.md]: I think we should go further here and recommend `openjdk/8-jre-alpine` as the base image to use.

(81.4MB in size)

File: src/main/play-doc/operation/Install.md [hunk 2]
```diff
@@ -247,15 +247,16 @@ or
 [172.17.0.1]$ sudo yum install conductr-agent-%PLAY_VERSION%-1.noarch.rpm
 ```
 
-then
+Next, you'll need to configure a couple of settings in `conductr-agent.ini` and grant some privileges. ConductR uses [runc](https://runc.io/) to spawn and run OCI-based bundles. This requires the OCI component of ConductR to have root access.
```
Comment (huntc) [inline src/main/play-doc/operation/Install.md]: Elaborate on "why", and also how we've mitigated the risk side given that the root privilege is only granted to the script.

**Review by huntc — CHANGES_REQUESTED** [review]

File: src/main/play-doc/developer/CreatingBundles.md (hunk 1 above)
Comment (longshorej) [reply src/main/play-doc/developer/CreatingBundles.md]: Thanks - that's set.

File: src/main/play-doc/operation/Install.md (hunk 2 above)
Comment (longshorej) [reply src/main/play-doc/operation/Install.md]: This is set.

**Review by huntc — APPROVED** [review]

**Pull request merged**